src/FSLibrary/StellarCoreCfg.fs

```diff
@@ -320,8 +320,9 @@ type StellarCoreCfg =
 
         let invList =
             match self.invariantChecks with
-            | AllInvariants -> [ ".*" ]
-            | AllInvariantsExceptBucketConsistencyChecks -> [ "(?!BucketListIsConsistentWithDatabase).*" ]
+            | AllInvariantsExceptEvents -> [ "(?!EventsAreConsistentWithEntryDiffs).*" ]
+            | AllInvariantsExceptBucketConsistencyChecksAndEvents ->
```

Comment:
Should we keep the `AllInvariants -> [ ".*" ]`. 
I understand are aren't using `AllInvariants` at the moment but this should be a valid option?

Reply:
We'd also have to allow for setting the events config flags, because `AllInvariants` wouldn't work without it. I'm not against that, but question if that's worth the time right now doing it when we need it.